### PR TITLE
Localize error messages

### PR DESCRIPTION
**Pull request by alicoding**
There are some missing and unlocalized strings in the repo

### CODE REVIEW

**Comment by alicoding** [comment]
@k88hudson r?

File: src/pages/login/form-input.jsx [hunk 1]
```diff
@@ -18,7 +18,11 @@ var FormInput = React.createClass({
     }
   },
   render: function () {
-    var helpText;
+    var helpText, errorMessage;
+
+    if (this.props.errors) {
+      errorMessage = this.getIntlMessage(this.props.errors.join(' '));
```
Comment (k88hudson) [inline src/pages/login/form-input.jsx:24]: don't you want to do something like

```
errorMessage = this.props.errors.map(this.getIntlMessage).join(' ');
```

File: src/pages/login/form-input.jsx [hunk 2]
```diff
@@ -18,7 +18,11 @@ var FormInput = React.createClass({
...
+    };
```
Comment (k88hudson) [inline src/pages/login/form-input.jsx:25]: remove this semicolon

**Comment by k88hudson** [comment]
just added a question and a small nit that is causing travis to fail

**Comment by alicoding** [comment]
Will probably need to rework this one again since there are lots of changes happened while I was away in SFO

**Pull request closed without merging**